### DISCUSSION TITLE
Copilot

### DISCUSSION THREAD

**Discussion by kurt-melony** [community/community#113453] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

What about your test subscription? I’m trying to register - it immediately bans my account, I’ve already paid $10, but they came back and I can’t make the payment again!

**Comment by kurt-melony** [comment]
@Akash1134

**Comment by davecheney** [comment, marked as answer]
@kurt-melony your credit card is being declined which prevents you from signing up for a Copilot plan. We cannot resolve billing issues via this forum, please contact support https://support.github.com/contact
